core/core.go

```diff
@@ -51,6 +51,9 @@ type Config struct {
 	// used for signing messages and generating a peer ID. If empty, a randomly
 	// generated key will be used.
 	PrivateKeyPath string `envvar:"PRIVATE_KEY_PATH" default:"./0x_mesh/key/privkey"`
+	// OrderExpirationBuffer is the number of seconds before the order's stipulated expiration time
+	// that you'd want it pruned from the Mesh node.
+	OrderExpirationBuffer int64 `envvar:"ORDER_EXPIRATION_BUFFER" default:"10"`
```

Comment:
`ExpirationBuffer` should probably have type `time.Duration` instead of `int64`. The env var package we're using supports any type which implements `TextMarshalerUnmarshaler`, so if we changed the type to `time.Duration`, it would accept values like `10s`, `1h`, or `2h20m`.

If you want to keep the env var as `int64` I'm okay with it, but at the very least we should convert to `time.Duration` after parsing the env var.

What do you think?

Reply:
Agreed! It actually lead to a necessary cleanup.